src/main/java/gg/solarmc/clans/SolarClans.java

```diff
@@ -63,6 +68,11 @@ public void onDisable() {
         LOGGER.info(String.format("[%s] Disabled Version %s", getDescription().getName(), getDescription().getVersion()));
     }
 
+    public MessageConfig getPluginConfig() {
+        config.reloadConfig();
```

Comment:
Every time you call this method, the config will be reloaded. That's a lot of I/O which will cause lag as players run commands.